File: game-utils/game-template/bots/java/src/core/MatchDetails.java
package core;

public class MatchDetails {

    /**
     * Index of the bot in botsDetails to which
     * this instance of MatchDetails was sent to
     */
    public int yourBotIndex;

    /**
     * List of all bots that participate in this
     * match with their details
     */
    public BotDetails[] botsDetails;
}

File: game-utils/game-example/bots/kotlin/src/core/MatchDetails.kt
package core

class MatchDetails(
        /**
         * Index of the bot in botsDetails to which
         * this instance of MatchDetails was sent to
         */
        public var yourBotIndex: Int,

        /**
         * List of all bots that participate in this
         * match with their details
         */
        public var botsDetails: Array<BotDetails>
)
